sysutils/lporg/Portfile

```diff
@@ -184,39 +184,46 @@ go.vendors          howett.net/plist \
+                        sha256  ab016c27522dde5d2554bd3d3e5f75ed2d8ed874b42c067717b3c83fe7eca3ca \
+                        size    560110 \
                     github.com/davecgh/go-spew \
                         lock    v1.1.1 \
                         rmd160  7c02883aa81f81aca14e13a27fdca9e7fbc136f7 \
                         sha256  e85d6afa83e64962e0d63dd4812971eccf2b9b5445eda93f46a4406f0c177d5f \
                         size    42171 \
+                    github.com/creack/pty \
+                        lock    v1.1.17 \
+                        rmd160  7bd32fd6c48511c8ce313e25d3a3100c8353460f \
+                        sha256  f11acd8470b37e5f6a1d9c810956ba700459eae0ff595566930b29d546f0245c \
+                        size    9978 \
                     github.com/cpuguy83/go-md2man \
-                        lock    v2.0.1 \
-                        rmd160  74c013e19d22e56a27d9a3ad61b4bd86975036d1 \
-                        sha256  d23365bceea7382b59ca41ad868a80e34f8066785fb371b85b51ee0b65be6a21 \
-                        size    64243 \
+                        lock    v2.0.2 \
+                        rmd160  23c11486c5bc6f87cb13d2cb2aa7c2c68fd28f96 \
+                        sha256  c0fe49af2717cef631621cbbf010c7ee69b7e5c8afcde33779e07ecece9c00cc \
+                        size    64382 \
                     github.com/apex/log \
                         lock    v1.9.0 \
                         rmd160  db8d16bcf0b7ea316e152eaef95aee474984ffb9 \
                         sha256  b098424ff812ffe08b754327356f22fc03a8053f4642a7b35f5262950523a222 \
                         size    31993 \
                     github.com/Netflix/go-expect \
-                        lock    c93bf25de8e8 \
-                        rmd160  a4e10dd1f05c584ae80408e373cc5b9e90581dc7 \
-                        sha256  896ec6b1f14446e88345be7dc7a8575957040b5a3729da1698ca88138e085ee2 \
-                        size    14580 \
+                        lock    73e0943537d2 \
+                        rmd160  acae8548d28c888cd16ade66df63a4bc2fa59bbc \
+                        sha256  81d106311b0a7a4971c45c14786451136bf238263fbb57d42977ecfd6f84a3a3 \
+                        size    15320 \
                     github.com/AlecAivazis/survey \
-                        lock    v2.3.2 \
-                        rmd160  4d4ffa6f9a7088f295ced08f60ba3a0bbbd39cb4 \
-                        sha256  5420c336cc99a5a92df0b24742f1217ab2e57ddeca497ca9ea689a2d2c5d4ff2 \
-                        size    1239289
+                        repo    github.com/go-survey/survey \
+                        lock    v2.3.6 \
+                        rmd160  8af519d0839cac947b44f67b037d930e41227690 \
+                        sha256  d692ec0dedfa6fdcd440c83829072cc3da69e6f2ddaa153f5a340f7f8d86fe0e \
+                        size    1466690
 
 patchfiles           patch-rm-buildtime-in-version.diff
 
 build.env-append    CGO_ENABLED=1
+build.env-delete    GO111MODULE=off GOPROXY=off
```

Comment:
Unsetting `GO111MODULE=off` and `GOPROXY=off` will completely remove the need for the entire `go.vendors` section.

`go.vendors` instructs MacPorts how to directly download the Go packages that `lporg` will need in order to build.

But if you unset `GO111MODULE=off` and `GOPROXY=off`, then Go will download those packages itself automatically as part of the build process, and ignore all the packages provided by `go.vendors`.

So if you are going to keep this line, you should remove the entire `go.vendors` block.

Reply:
updated